### PR TITLE
o added INSTALL.md

### PR DESCRIPTION
**Pull request by lromor**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+#uboot_overlay_pru=/lib/firmware/AM335X-PRU-UIO-00A0.dtbo
+###
+
+```
+
+uncommenting the line `uboot_overlay_pru=/lib/firmware/AM335X-PRU-UIO-00A0.dtbo`.
+
+Before rebooting, you will also need to have an updated version of your kernel as you may suffer a bug that will not correctly load the uio_pruss module.
+
+To do so, run `sudo -s -- apt-get update; apt-get upgrade` and on completion,
```
Comment (hzeller) [inline INSTALL.md]: this won't work, you need to issue sudo twice
 sudo apt-get update; sudo apt-get upgrade

Comment (lromor) [reply INSTALL.md]: You are right, sorry :) (solved)

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+# INSTALLATION
+
+## Get one of the latest linux Debian images
+
+Download one of the latest debian images provided by the following [**list**](https://beagleboard.org/latest-images):
+
+If necessary, check the integrity of the downloaded image by matching its sha256sum hash and the one provided on the website (on a linux terminal,
+just run `sha256sum bone-debian-XXXXXXXX.img.xz`).
+
```
[the remark this answers is not in the document]
Comment (lromor) [reply INSTALL.md]: (solved)

**Comment by hzeller** [comment]
Thanks, merged!

**Pull request merged**